src/bert_model.rs

```diff
@@ -654,6 +694,17 @@ mod tests {
         ]
     }
 
+    fn seqlen_to_mask(seq_lens: Tensor, max_len: i64) -> Tensor {
+        let batch_size = seq_lens.size()[0];
+        Tensor::arange(max_len, (Kind::Int, Device::Cpu))
```

Comment:
Does pinning the mask to CPU work when running on GPU?

Reply:
I don't know. Perhaps it would incur an extra transfer/copy each time? But this is only used for the unit test. If we need a crate-wide `seqlen_to_mask` at some point, I guess we should add the device as a parameter.

Reply:
Arf, my bad, for some reason I thought this was being used to create masks.

Reply:
Anyways, AFAIK, torch does not automatically copy tensors to the target device, in PyTorch things break if we pass a CPU tensor into a GPU model.